assets/src/scss/layout/navbar/languages.scss

```diff
@@ -12,11 +12,28 @@
     display: inline-block;
     line-height: $menu-height;
 
+    &::after {
+      background-color: var(--top-navigation--color, $white);
+      background-repeat: no-repeat;
+      content: "";
+      display: inline-block;
+      height: 8px;
+      margin-inline-start: 4px;
+      mask: url("../../images/chevron.svg") 0 0/8px 8px;
+      transform: rotate(90deg);
+      transition: transform 100ms linear;
```

Comment:
The transition feels very fast, from a quick look at the [navbar file](https://github.com/greenpeace/planet4-master-theme/blob/master/assets/src/scss/layout/_navbar.scss#L85) it seems for the dropdowns we use `150ms`, maybe that would be better?

Reply:
You're right! 
Also, I've changed with the same styles as the dropdown menu from the navigation bar that already has.